src/devices/honeywell_cm921.c

```diff
@@ -0,0 +1,449 @@
+/** @file
+    Honeywell CM921 Thermostat
+
+    Copyright (C) 2020
```

Comment:
There should be some author to the code.

Reply:
Done!